client/blocks/plugins-scheduled-updates-multisite/schedule-list-empty.tsx

```diff
@@ -0,0 +1,25 @@
+import { __experimentalText as Text, Button } from '@wordpress/components';
+import { plus } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+
+interface Props {
+	onCreateNewSchedule: () => void;
+}
+export const ScheduleListEmpty = ( props: Props ) => {
+	const translate = useTranslate();
+
+	const { onCreateNewSchedule } = props;
+
+	return (
+		<div className="empty-state">
+			<Text as="p">
+				{ translate(
+					"Take control of your site's maintenance by choosing when your plugins update—whatever day and time is most convenient. Enjoy hassle-free automatic updates, and our built-in rollback feature reverts any flawed updates for added peace of mind."
+				) }
+			</Text>
+			<Button __next40pxDefaultSize icon={ plus } variant="primary" onClick={ onCreateNewSchedule }>
+				{ translate( 'Setup a new schedule' ) }
```

Comment:
Any reason for not reusing the string we have in the "add schedule" button on single sites?

Reply:
Not particular, I was trying to follow the design here: rrEvrq63njWdmuXkJSTcY2-fi-3081_88566 . I can align this with the ones on the single sites.

Reply:
Thank you!